readme.md

```diff
@@ -143,7 +143,7 @@ UPDATE users SET email='stephanie_diaz@gmail.com' WHERE name='Stephanie';
     - One to One
         - Let's alter our example. Let's say our pictures table is actually a table of profile pictures. A user can only have one profile picture, and one profile picture can only have one user. 
         ```
-        CREATE TABLE users (
+        CREATE TABLE Users (
```

Comment:
You need to pull the latest changes from the upstream (my master branch) and push to your branch